apps/formbricks-com/pages/docs/getting-started/quickstart-in-app-survey/index.mdx

```diff
@@ -0,0 +1,136 @@
+
+<Image
+  src={I4}
+  alt="In app survey trigger for feedback popup micro survey"
+  quality="100"
+  className="rounded-lg"
+/>
+
+In **Recontact Options** we choose the following settings, so that we can play around with the survey more easily. By default, each survey will be shown only once to each user to prevent survey fatigue:
+
+<Image src={I5} alt="Options for survey popup in app micro survey" quality="100" className="rounded-lg" />
+
+Now hit **Publish** and you’ll be forwarded to the Summary Page. This is where you’ll find the responses to this survey. On the Summary Page click through to the Setup Checklist:
+
+<Image src={I6} alt="pop up survey settings for inapp web survey" quality="100" className="rounded-lg" />
+
+## 3. Set up the Formbricks Widget in your app
+
+On the Setup Checklist you have two elements. At the top you find the Widget Status Indicator. Once your app is connected to Formbricks Cloud successfully, this will turn green:
+
+<Image src={I7} alt="feedback popup in app survey" quality="100" className="rounded-lg" />
+
+In the manual below, this code snippet contains all the information you need:
+
+- The **environment ID** of your current Formbricks workspace
+- The **API Host** which is ‘https://app.formbricks.com’ for Cloud users
+
+<Image src={I8} alt="settings for in app survey popping up" quality="100" className="rounded-lg" />
+
+## Load Formbricks widget in your app
+
+In a local instance of your app, you'll embed the Formbricks Widget. Dependent on your frontend tech, the setup differs a bit:
+
+- [Next.JS App Directory](/docs/getting-started/nextjs-app)
+- [Next.JS Pages Directory](/docs/getting-started/nextjs-pages)
+- [React (Create React App)](/docs/getting-started/react)
+- [Vue.JS](/docs/getting-started/vuejs)
+
+## 4. Restart your app and open browser console
+
+Now, restart your app in your terminal to make sure the widget is loaded. Once its loaded, open the browser console to see the Formbricks debug logs. If you did everything right, you should now see your survey in the lower right corner:
+
+<Image
+  src={ReactApp}
+  alt="In app survey in React app for micro surveys"
+  quality="100"
+  className="rounded-lg"
+/>
+
+## 5. Most frequent errors and how to debug them
+
+In case you don’t see your survey right away, here's what you can do. Go through these in orderly to find the error fast:
+
+### A. Formbricks Cloud and your app are not connected properly.
+
+Go back to [app.formbricks.com](http://app.formbricks.com) and check go to the Setup Checklist in the Settings. If the status is still indicated as “Not connected” your app hasn't yet pinged the Formbricks Cloud:
+
+<Image
+  src={I9}
+  alt="setup checklist ui of survey popup for in app surveys"
+  quality="100"
+  className="rounded-lg"
+/>
+**How to fix it:**
+
+1. Check if your app loads the Formbricks widget correctly. Go to X, Y, Z
```

Comment:
X, Y, Z are placeholder? 😉

Reply:
No but I dont know the best approach for troubleshooting. Can you fill this in? 🙏